keyboards/converter/ibm_pingmaster/rules.mk

```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU               = atmega32u4
+
+# Bootloader selection
+BOOTLOADER        = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE  = lite      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE   = no        # Mouse keys
+EXTRAKEY_ENABLE   = yes       # Audio control and System control
+CONSOLE_ENABLE    = no        # Console for debug
+COMMAND_ENABLE    = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE  = no        # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE       = yes       # USB Nkey Rollover
+BACKLIGHT_ENABLE  = no        # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE   = no        # Enable keyboard RGB underglow
+UNICODE_ENABLE    = no        # Unicode
+BLUETOOTH_ENABLE  = no        # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE      = no        # Audio output on port C6
+CUSTOM_MATRIX     = yes
```

Comment:
Could this be converted to matrix lite? 

You can see some of the method to implement this here: 
https://docs.qmk.fm/#/custom_matrix?id=lite

Reply:
This is a port of the existing code written by Hasu.
I'll see what I can do about the matrix lite implementation.

Reply:
Matrix lite doesn't exist in tmk.   And the point of it is that you don't have re-implement the entire matrix.  Just the scan itself, and the initialization.   This way you don't have to worry about implementing debouncing and other such.    So, it's cleaner and simpler.